openpype/hosts/nuke/plugins/load/load_effects.py

```diff
@@ -156,7 +156,7 @@ def update(self, container, representation):
         """
         # get main variables
         # Get version from io
-        project_name = legacy_io.active_project()
+        project_name = get_current_asset_name()
```

Comment:
This is refactored wrong. Should be `get_current_project_name`
```suggestion
        project_name = get_current_project_name()
```